docs/UserGuide.md

```diff
@@ -132,18 +132,11 @@ Note: Address cannot be empty. It must contain at least 1 character.
 Note: Email cannot be empty. It must fulfil the above requirements.
 ```
 
-6. Class Date:
-    - Class Date must be in the format YYYY-MM-DD {start time}-{end time}.
-
-```yaml
-Note: Start time and End time must be in 24hour format.
-```
-
-Format: `add n/NAME p/CONTACT_NUMBER np/NEXT_OF_KIN_CONTACT_NUMBER a/ADDRESS e/EMAIL dt/CLASS_DATE`
+Format: `add n/NAME p/CONTACT_NUMBER np/NEXT_OF_KIN_CONTACT_NUMBER e/EMAIL a/ADDRESS`
```

Comment:
I was thinking about removing np/ as well, as our v1.2 doesn't support it. Not sure whether the UG is supposed to be updated for v1.2.